packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1839,27 +1841,13 @@ export class AccountHandler {
       }
     }
 
-    const { uid } = accountRecord;
-
-    // We fetch the devices to notify before deleteAccount()
-    // because obviously we can't retrieve the devices list after!
-    const devices = await this.db.devices(uid);
-    const push = this.push;
-    const log = this.log;
-
-    const notify = async () => {
-      try {
-        log.info('accountDeleted.byRequest', accountRecord as Account);
-        await push.notifyAccountDestroyed(uid, devices);
-        await log.notifyAttachedServices('delete', request, { uid });
-        await request.emitMetricsEvent('account.deleted', { uid });
-      } catch (error) {
-        log.error('Account.destroy.notify', { uid, error });
-      }
-    };
-
-    await this.accountDeleteManager.deleteAccount(uid, '', { notify });
-
+    await this.accountDeleteManager.quickDelete(
+      accountRecord.uid,
+      ReasonForDeletion.UserRequested
+    );
```

Comment:
This will throw an error in any environment that doesn't have cloud tasks set up to accept the new task.

Reply:
Ah, we should error on startup if cloud tasks is not configured as this is an important requirement here.

Reply:
Added a check so its only required in stage/prod or if its configured.